aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -152,6 +156,16 @@ func resourceAwsApiGatewayRestApiRead(d *schema.ResourceData, meta interface{})
 	d.Set("name", api.Name)
 	d.Set("description", api.Description)
 	d.Set("binary_media_types", api.BinaryMediaTypes)
+
+	restApiId := d.Id()
+	region := meta.(*AWSClient).region
+	accountId := meta.(*AWSClient).accountid
+	arn, err := buildApiGatewayExecutionARN(restApiId, region, accountId)
```

Comment:
This function no longer exists as we have moved to prefer using the SDK `arn` package. Can you please build the ARN like the following (modifying as appropriate):

```go
executionArn := arn.ARN{
		Partition: meta.(*AWSClient).partition,
		Service:   "execute-api",
		Region:    meta.(*AWSClient).region,
		AccountID: meta.(*AWSClient).accountid,
		Resource:  fmt.Sprintf("%s/%s", restApiId, stageName),
	}.String()
	d.Set("execution_arn", executionArn)
```